sdk-setup/src/mb2

```diff
@@ -2688,7 +2688,10 @@ init_patch_wrapper() (
     local patchset_id=$2
 
     # Use git from host
-    git() { SBOX_DISABLE_MAPPING=1 /usr/bin/git "$@"; }
+    git() {
+        local original_git=$(which git)
```

Comment:
The `which` command needs to be used on host to be always accurate, so we need

```
HOST_GIT=$(quote "$(which git)")
```

below (preferably just above the `WE=$(quote "$WE")` line https://github.com/sailfishos/sdk-setup/pull/352/files#diff-ae2d3f30654b05a59e4f75cd131205c52ce1b713c879047de93d8965384ca516R2839) and then use `$HOST_GIT` in place of `$original_git` here.

Reply:
That didn't quite work, as git() is used already in init_patch_wrapper, and not just in the wrapper. Also, while applying your proposed change, I noticed that init_commit_proposed_changes_hook needs the same change as well.